department-of-veterans-affairs/caseflow
# Queue | Display info for multiple hearings

Resolves #4403

### Description
Hearings section of Appeal Detail tab now displays information for multiple hearings, if applicable.

### Acceptance Criteria 
- [ ] Verify that each hearing for an appeal has corresponding data for date held and name of judge at hearing

### Testing Plan
(Enable `:queue_welcome_gate` and `:queue_case_search` feature toggles)
1. Go to `/queue`, search for `760362568S` for an appeal with 1 hearing, `228081153S` for 2 hearings.
2. Verify `Hearings (Oldest to Newest)` is displayed when > 1 hearing, hidden when 1 hearing.
3. Verify that `type`, `disposition`, `date`, and `judge` are displayed for each hearing

1 hearing
![screen shot 2018-05-14 at 4 17 51 pm](https://user-images.githubusercontent.com/8533004/40021282-5ef6d9ac-5792-11e8-8f2c-a2a033e435aa.png)


2 hearings
![screen shot 2018-05-14 at 4 17 17 pm](https://user-images.githubusercontent.com/8533004/40021255-51d81e2a-5792-11e8-98ac-11ee91587175.png)
